server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2847,13 +2847,18 @@ public void clusterStateProcessed(String source, ClusterState oldState, final Cl
         assertAcked(client.admin().cluster().prepareDeleteSnapshot(repositoryName, snapshotName).get("10s"));
     }
 
-    private boolean waitForIndex(final String index, TimeValue timeout) throws InterruptedException {
-        return awaitBusy(() -> indexExists(index), timeout.millis(), TimeUnit.MILLISECONDS);
+    private void waitForIndex(final String index, TimeValue timeout) throws Exception {
+        assertBusy(
+            () -> assertTrue("Expected index [" + index + "] to exist", indexExists(index)),
+            timeout.millis(),
+            TimeUnit.MILLISECONDS);
     }
 
-    private boolean waitForRelocationsToStart(final String index, TimeValue timeout) throws InterruptedException {
-        return awaitBusy(() -> client().admin().cluster().prepareHealth(index).execute().actionGet().getRelocatingShards() > 0,
-            timeout.millis(), TimeUnit.MILLISECONDS);
+    private void waitForRelocationsToStart(final String index, TimeValue timeout) throws Exception {
```

Comment:
The called of this method never checked its return value. Now that the method is calling `assertBusy` instead of `awaitBusy`, the test fails, because the number of relocating shards is not greater than zero.

I'm not sure who to ping about this - @elastic/es-distributed maybe?